src/components/InvertedFlatList/index.tsx

```diff
@@ -77,9 +56,8 @@ function InvertedFlatList(props) {
      * This workaround is taken from below and refactored to fit our needs:
      * https://github.com/necolas/react-native-web/issues/1021#issuecomment-984151185
      *
-     * @param {Event} event - The onScroll event from the FlatList
```

Comment:
Let's keep the comment since it has a description, only remove the `{Event}` part